source/cursorManager.py

```diff
@@ -59,8 +62,33 @@ def __init__(self, parent, cursorManager, text, caseSensitivity):
 		self.CentreOnScreen()
 		self.findTextField.SetFocus()
 
+	def updateSearchEntries(self, searchEntries, currentSearchTerm):
+		if not currentSearchTerm:
+			return
+		if not searchEntries:
+			searchEntries.insert(0, currentSearchTerm)
+			return
+		# we can not accept entries that differ only on text case because of a wxComboBox limitation on MS Windows
+		# see https://wxpython.org/Phoenix/docs/html/wx.ComboBox.html
+		#notice also that python 2 does not offer caseFold functionality so lower is the best we can have for comparing strings
+		for index, item in enumerate(searchEntries):
+			if(item.lower() == currentSearchTerm.lower()):
+				#if the user has selected a previous search term in the list or retyped an already listed term , we need to make sure the current search term becomes the first item of the list,

```

Comment:
Please adjust the length of these comments to no more than 80-100 characters. Long lines are hard to read for sighted developers because they are partially hidden.